src/herder/TransactionQueue.h

```diff
@@ -143,6 +144,15 @@ class TransactionQueue
                                     std::vector<TransactionFramePtr>>;
     // keepBacklog: keeps transactions succeding tx in the account's backlog
     ExtractResult extract(TransactionFramePtr const& tx, bool keepBacklog);
+
+    // size of the transaction queue, in operations
+    size_t mQueueSizeOps{0};
```

Comment:
You don't have any tests checking that this is being maintained properly across bans, invalid transactions, and applied transactions.

Reply:
done